styx-service-common/src/main/java/com/spotify/styx/storage/AggregateStorage.java

```diff
@@ -61,6 +61,7 @@ public AggregateStorage(Connection connection, Datastore datastore) {
 
   @Override
   public void close() throws IOException {
+    bigtableStorage.close();
```

Comment:
In `AggregateStorageTest` there is a test for `shouldCloseDatastore` but no for BigTable